libraries/chain/asset_evaluator.cpp

```diff
@@ -366,16 +366,16 @@ void_result asset_update_issuer_evaluator::do_apply(const asset_update_issuer_op
  *
  * @param d the database
  * @param op the bitasset update operation being performed
+ * @param new_backing_asset
  * @param true if after hf 922/931 (if nothing triggers, this and the logic that depends on it
  *    should be removed).
  */
-void check_children_of_bitasset(database& d, const asset_update_bitasset_operation& op, bool after_hf_922_931)
+void check_children_of_bitasset(database& d, const asset_update_bitasset_operation& op,
+      const asset_object& new_backing_asset, bool after_hf_922_931)
 {
```

Comment:
To be protective, when `new_backing_asset` is CORE, return.
That said, when changing an MPA to be backed by CORE, no need to check children.
Actually this function will only be called when `new_backing_asset` is not CORE.

Reply:
Added check to be sure